### ISSUE TITLE
Implementar Mapbox Clusters

### ISSUE THREAD

**Issue by tamielbr** [gabrielfedel/CiclopeDB#46]
Alguns estados, como NEaDs no RJ, poderiam se beneficiar do cluster. Ficam muito juntos e é difícil de clicar mesmo no zoom máximo.

https://github.com/Leaflet/Leaflet.markercluster

**Comment by kalimar** [comment]
https://www.mapbox.com/mapbox-gl-js/example/cluster/

**Comment by greevin** [comment]
Adicionei os clusters para os IPES com o clusterRadius em 30, com isso ele consegue juntar bem os números dos estados.

![sem titulo](https://user-images.githubusercontent.com/7833013/38094643-b8b5fb02-3344-11e8-9e53-e8f524b06f60.png)

**Comment by tamielbr** [comment]
@greevin só foi IPES por enquanto né?

**Comment by greevin** [comment]
@tamielbr Sim. Pelo que eu andei vendo, se eu colocar dois clusters (mesmo com duas layers diferentes), ele entende como uma só.

![cas](https://user-images.githubusercontent.com/7833013/38195718-5fe0cd40-3655-11e8-856f-81857200a67b.png)

**Comment by tamielbr** [comment]
Precisamos avaliar o uso dos clusters com a usabilidade e visualização e tecnicamente, compatibilidade com módulo de busca.
